### PR TITLE
enhancements as per UI feedback

### PR DESCRIPTION
**Pull request by pooja-koul-incontact**

### CODE REVIEW

File: src/app/bu/bu-user-mapping/bu-manual-mapping/bu-manual-mapping.component.spec.ts [hunk 1]
```diff
@@ -15,18 +15,25 @@ describe('BuManualMappingComponent', () => {
...
-  });
+  }));
+  beforeAll((async () => {
+    const row = {
+      'firstName': 'Administrator' ,
+      'lastName': 'tc3.com',
+      'email': 'firstName.lastName@nice.com',
+      'agentId': '287956'
+    };
+    localStorage.setItem('selectedRow', JSON.stringify(row));
```
Comment (saiprasad1997) [inline src/app/bu/bu-user-mapping/bu-manual-mapping/bu-manual-mapping.component.spec.ts]: no need to store this in local storage for unit testing

File: src/app/bu/bu-user-mapping/bu-user-mapping-status/bu-user-mapping-status.component.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.completed-icon-to-display {
```
Comment (saiprasad1997) [inline src/app/bu/bu-user-mapping/bu-user-mapping-status/bu-user-mapping-status.component.css]: add container name

File: src/app/user-mapping/manual-mapping/manual-mapping.component.spec.ts [hunk 3]
```diff
@@ -18,15 +18,22 @@ describe('BuManualMappingComponent', () => {
...
-  });
+  }));
+  beforeAll((async () => {
+    const row = {
+      'firstName': 'Administrator' ,
+      'lastName': 'tc3.com',
+      'email': 'firstName.lastName@nice.com',
+      'agentId': '287956'
+    };
+    localStorage.setItem('selectedRow', JSON.stringify(row));
```
Comment (saiprasad1997) [inline src/app/user-mapping/manual-mapping/manual-mapping.component.spec.ts]: no need to use local storage for unit test cases

File: src/app/user-mapping/user-mapping-status/user-mapping-status.component.css [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.completed-icon-to-display {
```
Comment (saiprasad1997) [inline src/app/user-mapping/user-mapping-status/user-mapping-status.component.css]: add container name

File: src/app/bu/bu-user-mapping/bu-saved-mapping/bu-saved-mapping.component.css [hunk 5]
```diff
@@ -3,7 +3,7 @@
   font-size: 15px;
   padding-left: 30px;
   padding-right: 30px;
-  font-family: 'Open Sans';
+  font-family: 'OpenSans';
```
Comment (KMayuri25) [inline src/app/bu/bu-user-mapping/bu-saved-mapping/bu-saved-mapping.component.css]: no need to specify font again

File: src/app/bu/bu-user-mapping/bu-user-mapping-status/bu-user-mapping-status.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ BuUserMappingStatusComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(BuUserMappingStatusComponent);
```
Comment (KMayuri25) [inline src/app/bu/bu-user-mapping/bu-user-mapping-status/bu-user-mapping-status.component.spec.ts]: use async

**Pull request merged**